packages/apps/explorer/src/components/search/search.css.ts

```diff
@@ -0,0 +1,40 @@
+import { atoms, responsiveStyle } from '@kadena/react-ui/styles';
+import { style } from '@vanilla-extract/css';
+
+export const searchBoxClass = style({
+  ...responsiveStyle({
+    md: {
+      width: 525,
```

Comment:
use the tokens from react-ui for these values

Reply:
I dont think there are tokens with these values. And I can't seem to apply mathematical operations to the tokens (otherwise I could use a token's value and manipulate it to obtain the desired value)